virttest/qemu_vm.py

```diff
@@ -2302,11 +2293,17 @@ def sort_key(dev):
             # disable-legacy and disable-modern options explicitly.
             set_disable_legacy = params.get("set_disable_legacy", "no")
             set_disable_modern = params.get("set_disable_modern", "no")
+            set_iommu_platform = params.get("set_iommu_platform", "no")
+            set_ats = params.get("set_ats", "no")
             if dev_type in virtio_pci_devices:
                 if set_disable_legacy == "yes":
-                    add_disable_legacy(devices, device, dev_type)
+                    add_virtio_option("disable-legacy", "on",  devices, device, dev_type)
```

Comment:
default value for disable-legacy should be off, right? refer line 1312

Reply:
@CongLi  based on machine type, if you run 440fx, the default value is off

Reply:
@CongLi   L1312 is the old value.   We use virtio 1.0 by default now,  so disable-legacy is on by default.

And this change will not effect other cases unless  "set_disable_legacy" be set to "yes".

Reply:
Hi guys,

here just need to care code level default values,  default value for  different machine type is should been control by machine.cfg

Thanks,
Xu

Reply:
Thanks @xutian . yes, and  i does not distinguish the default value for different machine type here.
Only one default value here.